src/components/LoginPage.jsx

```diff
@@ -0,0 +1,143 @@
+import { Formik, Form, Field, ErrorMessage } from "formik";
+import { Container, Button, Image, Col } from "react-bootstrap";
+import { Link, Navigate } from "react-router-dom";
+import * as Yup from "yup";
+import bcrypt from "bcryptjs";
+import { useDispatch, useSelector } from "react-redux";
+import { LOGIN_USER } from "../redux/constants";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+const LoginPage = () => {
+  const usersData = useSelector((state) => state.usersReducer.usersData);
+  const isLoggedIn = useSelector((state) => state.usersReducer.loginUserData);
+  const dispatch = useDispatch();
+
+  const onSubmit = (values, { resetForm }) => {
+    values.password = bcrypt.hashSync(
+      values.password,
+      "$2a$10$CwTycUXWue0Thq9StjUM0u"
+    );
+    let loggedInUser = usersData.find((user) => user.email === values.email);
+    if (loggedInUser) {
+      if (loggedInUser.password === values.password) {
+        dispatch({
+          type: LOGIN_USER,
+          payload: loggedInUser,
+        });
+      } else {
+        toast.error("Incorrect Password", {
+          position: "top-right",
+          autoClose: 2000,
+          hideProgressBar: false,
```

Comment:
A lot of toast props are repeated in several places. You should store them in the object and reuse them.